packages/apollo-gateway/src/QueryPlan.ts

```diff
@@ -1,28 +1,28 @@
-import {
+  import {
```

Comment:
Unintentional indent?

```suggestion
import {
```